lib/workspace.js

```diff
@@ -124,6 +144,7 @@ module.exports = {
         const postJson = {
             workspace_id: workspace_id,
             action: action,
+            useInitialZip: state == 'uninitialized',
```

Comment:
@mwest1066 Or should I be passing in `useInitialZip` on each `controlContainer` call (that also feels a bit odd since `useInitialZip` only applies to the `init` action)? Can we determine `useInitialZip` using on the `state` here, even for to-be-implemented actions?

Reply:
I think we should not pass `state` to `controlContainer`. Instead I suggest adding an `options` argument like this: `controlContainer(workspace_id, action, options={})`. The `options` value should be an object. Inside `controlContainer` we can do:
```
        const postJson = {
            workspace_id: workspace_id,
            action: action,
            ...options,
        };
```

Reply:
This means that we can either do `controlContainer(workspace_id, 'getGradedFiles')` or `controlContainer(workspace_id, 'init', {useInitialZip: true})`.

Reply:
And in `interface.js` for the `init` handler we can do `const useInitialZip = _.get(options, 'useInitialZip', false)` to get it with a default value.